plugins/modules/nmcli.py

```diff
@@ -80,6 +82,13 @@
         type: str
         choices: [ datagram, connected ]
         version_added: 5.8.0
+    slave_type:
+        description:
+            - Type of the device of this slave's master connection (eg, "bond").
```

Comment:
```suggestion
            - Type of the device of this slave's master connection (for example C(bond)).
```

Reply:
Fixed in [858b601](https://github.com/ansible-collections/community.general/pull/6108/commits/858b6011503d524aef7016d6640ee4b2627c70cd)